Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -8,28 +8,29 @@
 
 namespace UnityEngine.InputSystem.Editor
 {
+    class ProjectWideActionsAssetPostprocessor : AssetPostprocessor
+    {
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths, bool didDomainReload)
+        {
+            if (ArrayHelpers.Contains(importedAssets, ProjectWideActionsAsset.kAssetPath))
```

Comment:
Does this imply that the .additionalfile has a fixed path? Wouldn't it be possible to instead scan for it or have its location tracked by this and persisted as part of InputManager.asset scriptable object or as an optional serialised property of the InputActionAsset updated via this if an importer cannot be registered?